frontend/sac-mobile/app/(auth)/login.tsx

```diff
@@ -12,7 +12,7 @@ const Login = () => {
             <View className="flex-1">
                 <View className="px-[8%] pb-[10%]">
                     <View className="pt-[1%]">
-                        <Wordmark />
+                        <Wordmark textColor="text-white" />
```

Comment:
instead of taking a specific color which is too granular, name specific types such as primary, secondary, default